packages/react-next/src/common/_constants.scss

```diff
@@ -10,3 +10,5 @@ $animationDuration1: 0.167s;
 $animationDuration2: 0.267s;
 $animationDuration3: 0.367s;
 $animationDuration4: 0.467s;
+
+$CONTEXTUAL_MENU_ITEM_HEIGHT: 36px;
```

Comment:
Can this value be influenced by the theme or the size of the pivot item?

Reply:
This is just copying a hardcoded value in the old Fabric ContextualMenu code (I'll add a comment about that), so it can't be changed for now. Eventually Pivot will be moved to use the new Button and ContextualMenu; I'm not sure if the new ContextualMenu will support theming this value.